tools/opt/size-passes.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPIRV_TOOLS_SIZE_PASSES_H_
+#define SPIRV_TOOLS_SIZE_PASSES_H_
+
+#include <vector>
+#include <string>
+
+namespace spvtools {
+
+// Size optimization flags expanded by -Os.
+//
+// Using the -Os flag in spirv-opt's command line is equivalent to specifying
+// the following flags in this exact sequence.
```

Comment:
Recommend a note this list is subject to continual tuning.

Reply:
Done.